src/logger/lwt_log_core.mli

```diff
@@ -86,7 +86,9 @@ type section
 
 val string_of_level : level -> string
 
-val load_rules : string -> unit
+val level_of_string : string -> (level, string) result
+
+val load_rules : ?fail_on_error:bool -> string -> unit
```

Comment:
Should probably document `?fail_on_error` as, besides the exception, there is the non-trivial difference that `load_rules ~fail_on_error:false` will actually load as many rules as it can, while `load_rules ~fail_in_error:true` will load nothing if any error is encountered.